baseband/vlbi_base/base.py

```diff
@@ -253,10 +253,18 @@ def tell(self, unit=None):
         return (self.offset / self.sample_rate).to(unit)
 
     def _frame_info(self):
-        offset = (self.offset +
-                  self.header0['frame_nr'] * self.samples_per_frame)
-        framerate = int(np.round(
-            (self.sample_rate / self.samples_per_frame).to_value(u.Hz)))
+        # Can be made less unwieldy if we abstract a StreamBase class such that
+        # VLBIStreamBase is only for VLBI formats.
+        try:
+            framerate = self._framerate
+            offset0 = self._offset0
+        except AttributeError:
+            self._framerate = int(np.round(
```

Comment:
Any reason not to do `framerate = self._framerate = ...` -- it is standard python! (But obviously not a big deal at all)

Reply:
Really!  I'll change it.